content/rule_book/internationals.md

```diff
@@ -33,7 +33,7 @@ These rules apply to the Internationals quiz meet conducted once yearly, adminis
 
 ### Scorekeeper
 
-- At least one scorekeeper will tabulate the results of each question on official score sheets in every International Final. They shall keep a running score of each quiz.
+- At least one scorekeeper will tabulate the results of each question on official scoresheets in every International Final. They shall keep a running score of each quiz.
```

Comment:
every reference to singular "scoresheet" is a single word.

Reply:
Technically this is contrary to [dictionary.com](https://www.dictionary.com/misspelling?term=scoresheet), but I don't care much, as long as we're universally consistent.

Reply:
Roll with no spaces, since "scoresheet" is used liberally in ES?

Reply:
Cool. Agreed.